apps/antalmanac/src/stores/Schedules.ts

```diff
@@ -50,7 +51,7 @@ export class Schedules {
      * @return a specific schedule name
      */
     getScheduleName(scheduleIndex: number) {
-        return this.schedules[scheduleIndex].scheduleName;
+        return this.schedules[scheduleIndex]?.scheduleName;
```

Comment:
This is is the actual fix right? Everything else was just a refactor to make writing the test work?

Reply:
Yes, I just wanted to add a question mark and a test for that lol